frontend/src/fetchers/ssdi.js

```diff
@@ -44,10 +44,25 @@ const allowedTopicsForQuery = {
     'createDate',
   ],
   'qa-dashboard': [...QA_DASHBOARD_FILTER_CONFIG.map((filter) => filter.id),
+    'endDate',
+    'startDate',
+    'activityReportGoalResponse',
+    'goalName',
+    'grantNumber',
+    'group',
+    'singleOrMultiRecipients',
+    'programType',
+    'reason',
+    'recipient',
     'region',
     'reportId',
-    'activityReportGoalResponse',
+    'reportText',
     'role',
+    'stateCode',
+    'targetPopulations',
+    'topic',
+    'ttaType',
+    'status',
```

Comment:
When lists get this long I think it's good to sort them. Also prevents it from looking like an element was deleted in a diff when it was really just moved.

Reply:
This is still a temporary measure as the ssdi system has an endpoint to give all the filters it supports that should be used when we get some breathing room instead of having it statically defined.